htdocs/emailcollector/class/emailcollector.class.php

```diff
@@ -989,6 +989,7 @@ private function overwritePropertiesOfObject(&$object, $actionparam, $messagetex
 								}
 							}
 						}
+						$substitutionarray['__DATE_NOW__'] = dol_now();
```

Comment:
Why not calling getCommonSubstitutionArray() so the substitution array will contains all other entries. The dol_now will be available with __NOW_TMS__

Reply:
Yes I will do as you suggest to me

Reply:
@eldy 
I used getCommonSubstitutionArray function as you suggested and removed __DATE_NOW__ substitute key.